CPAN/buildme.sh

```diff
@@ -1575,13 +1563,7 @@ if [ $PERL_MINOR_VER -ge 12 ]; then
     ARCH=`$PERL_BIN -MConfig -le 'print $Config{archname}' | sed 's/gnu-//' | sed 's/^i[3456]86-/i386-/' | sed 's/armv.*?-/arm-/' `
 fi
 mkdir -p $PERL_ARCH/$ARCH
-rsync -amv --include='*/' --include='*.so' --include='*.bundle' --include='autosplit.ix' --exclude='*' $PERL_BASE/lib/perl5/*/auto $PERL_ARCH/$ARCH/
-if [ -n "${buildIOSocketSSL}"  ]; then
-    rsync -amv --include='*/' --include='Socket/' --include='*.pm' --exclude='*' $PERL_BASE/lib/perl5/IO $PERL_ARCH/$ARCH/
-    rsync -amv --include='*/' --include='IDN' --include='*.pm' --exclude='*' $PERL_BASE/lib/perl5/Net $PERL_ARCH/$ARCH/
```

Comment:
Don't know what this item "IDN" is. But are you sure it's covered by your single-liner?

Reply:
Woops. You're right. Corrected with a rebase. Now everything within perl5/lib is replicated into $PERL_ARCH/$ARCH.

Reply:
I don't think your change made it to the pull request?